<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>791</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="MaingridLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1286</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="open_file"/>
    <addaction name="new_file"/>
    <addaction name="close_file"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addWinAction"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addWinAction2"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addWinAction3"/>
  </widget>
  <widget class="QToolBar" name="toolBar_4">
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addWinAction4"/>
  </widget>
  <action name="open_file">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="new_file">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="close_file">
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="addWinAction">
   <property name="text">
    <string>图像处理</string>
   </property>
   <property name="toolTip">
    <string>图像处理</string>
   </property>
  </action>
  <action name="addWinAction2">
   <property name="text">
    <string>音频处理</string>
   </property>
   <property name="toolTip">
    <string>音频处理</string>
   </property>
  </action>
  <action name="addWinAction3">
   <property name="text">
    <string>视频取帧</string>
   </property>
   <property name="toolTip">
    <string>视频取帧</string>
   </property>
  </action>
  <action name="addWinAction4">
   <property name="text">
    <string>数据压缩</string>
   </property>
   <property name="toolTip">
    <string>数据压缩</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
